docs/release-notes.md

```diff
@@ -45,6 +45,7 @@ nav_order: 9
 - Drop `LuksOption` and `RaidOption` types _(Go API for fcos 1.5.0+,
   flatcar 1.1.0+, openshift 4.14.0-experimental)_
 - Require Go 1.18+
+- test: Add a test for fcos/v1_6_exp to detect bad partition table configurations when wipe_tables is false and the partition table is not on the root disk.
```

Comment:
Firstly; great attempt! 
So with our release notes (similarly to our commit messages) there are a few things to consider. 

You want to be concise as this is going to be a bulleted list at release time. (i.e https://github.com/coreos/butane/releases)

You also want to ask what part of the added work impacts the end user, if any (non developer)?
In this work example, the end user would be impacted more by the warning as they could see it. As the tests would only directly effect our developers working on the project. 

So we might want to try another swing at this with the context that this is a broadcast the the end user, and they would be reading this for what functionality was gained or lost since the last release.

Reply:
pushed a rewording of it lmk what you think!